boards/BMS/Src/App/App_TractiveSystem.c

```diff
@@ -51,3 +51,8 @@ float App_TractiveSystem_GetLowResCurrent(struct TractiveSystem *ts)
 {
     return ts->get_low_res_current(ts->get_adc_low_res_current());
 }
+
+float App_TractiveSystem_GetPower(struct TractiveSystem *ts)
+{
+    return App_TractiveSystem_GetVoltage(ts) * App_TractiveSystem_GetLowResCurrent(ts);
```

Comment:
I understand we will not always be in the low res current range. Should come with a function that decides when to return the low res or high instead of letting the user decide?

Reply:
yeah I can create a PR for changes to the current sensing